lib/vtls/vtls.c

```diff
@@ -737,9 +737,9 @@ CURLcode Curl_ssl_initsessions(struct Curl_easy *data, size_t amount)
     return CURLE_OUT_OF_MEMORY;
 
   /* store the info in the SSL section */
-  data->set.general_ssl.max_ssl_sessions = amount;
+  data->state.session_size = amount;
   data->state.session = session;
-  data->state.sessionage = 1; /* this is brand new */
+  data->state.sessionage = 0; /* this is brand new */
```

Comment:
I don't follow, why this change?

Reply:
In ``curl_share_setopt()`` (share.c), the initialization goes like this:
```
share->sessionage = 0;
```
I wanted to use the same value in vtls.c and in share.c. Alternatively share.c could be changed:
```
share->sessionage = 1;
```